bitwarden_license/bit-web/src/app/secrets-manager/layout/header.component.html

```diff
@@ -1,56 +1,96 @@
-<div class="tw-mb-3 tw-flex tw-items-center tw-gap-2" *ngIf="routeData$ | async as routeData">
-  <h1 class="tw-m-0 tw-mr-2 tw-text-3xl tw-font-semibold">{{ routeData.title | i18n }}</h1>
-  <div class="tw-ml-auto tw-w-1/4">
-    <input bitInput class="search tw-w-full" placeholder="{{ routeData.searchTitle | i18n }}" />
-  </div>
-  <sm-new-menu></sm-new-menu>
-  <product-switcher></product-switcher>
-  <ng-container *ngIf="account$ | async as account">
-    <button [bitMenuTriggerFor]="accountMenu" class="tw-border-0 tw-bg-transparent tw-p-0">
-      <bit-avatar [id]="account.userId" [text]="account.name || account.email"></bit-avatar>
-    </button>
+<header
+  *ngIf="routeData$ | async as routeData"
+  class="-tw-m-6 tw-mb-3 tw-flex tw-w-full tw-flex-col tw-p-6"
+  [ngClass]="{
+    'tw-border-0 tw-border-b tw-border-solid tw-border-secondary-300 tw-bg-background-alt tw-pb-0':
+      tabsContainer.childElementCount !== 0
+  }"
+>
+  <div class="tw-flex tw-w-full">
+    <div class="tw-flex tw-w-full tw-min-w-0 tw-flex-1 tw-flex-col tw-gap-2">
+      <ng-content select="[slot=breadcrumbs]"></ng-content>
+      <div #titleContainer [ngClass]="{ 'tw-hidden': titleContainer.childElementCount === 0 }">
+        <ng-content select="[slot=title]"></ng-content>
+      </div>
+      <h1
+        *ngIf="titleContainer.childElementCount === 0"
+        class="tw-m-0 tw-mr-2 tw-min-w-0 tw-truncate tw-text-3xl tw-font-semibold tw-leading-10"
+        [title]="title || (routeData.titleId | i18n)"
+      >
+        <i *ngIf="icon" class="bwi {{ icon }}" aria-hidden="true"></i>
+        {{ title || (routeData.titleId | i18n) }}
+      </h1>
+    </div>
```

Comment:
We can remove some of the widths, which also fixes a defect where the header didn't expand all the way due to the negative margin.

```suggestion
<header
  *ngIf="routeData$ | async as routeData"
  class="-tw-m-6 tw-mb-3 tw-flex tw-flex-col tw-p-6"
  [ngClass]="{
    'tw-border-0 tw-border-b tw-border-solid tw-border-secondary-300 tw-bg-background-alt tw-pb-0':
      tabsContainer.childElementCount !== 0
  }"
>
  <div class="tw-flex">
    <div class="tw-flex tw-min-w-0 tw-flex-1 tw-flex-col tw-gap-2">
      <ng-content select="[slot=breadcrumbs]"></ng-content>
      <div #titleContainer [ngClass]="{ 'tw-hidden': titleContainer.childElementCount === 0 }">
        <ng-content select="[slot=title]"></ng-content>
      </div>
      <h1
        *ngIf="titleContainer.childElementCount === 0"
        class="tw-m-0 tw-mr-2 tw-truncate tw-text-3xl tw-font-semibold tw-leading-10"
        [title]="title || (routeData.titleId | i18n)"
      >
        <i *ngIf="icon" class="bwi {{ icon }}" aria-hidden="true"></i>
        {{ title || (routeData.titleId | i18n) }}
      </h1>
    </div>
```